### DISCUSSION TITLE
Mastodon URL Redirect Help

### DISCUSSION THREAD

**Discussion by pumpkinlawrence** [mastodon/mastodon#23133] [Q&A]
I have an instance that is at xyz.com. In order to access the instance, I type xyz.com in the browser and it will load the site correctly. However, if I try to  type www.xyx.com, I get the NGINX welcome page. How do I make it so that when someone types in the xyz.com, it'll redirect to www.xyz.com. 

I have searched all over the answer. I thought it had something to do with enabling WEB_ROOT, but I think this can be done in NGINX. 

I am a newbie at this. Please help me. 

Thank you.

**Comment by mtbluedog** [comment]
Take a look at https://stackoverflow.com/questions/51473035/how-can-i-redirect-non-www-to-www-in-https-nginx
